src/get-started/java-binding.md

```diff
@@ -1,8 +1,73 @@
 # Java Binding User Guide
 
-**Java Binding App** is an application that includes the Exonum framework
+**Exonum Java App** is an application that includes the Exonum framework
 and Java services runtime environment.
 
+## Installation
+
+To run a node with your Java service you need to use Exonum Java application.
+You can download an archive containing the application and all the necessary
+dependencies on [GitHub releases page][github-releases]. We suggest to use
```

Comment:
> on the Releases page on Github

otherwise it seems like there is a page of all releases that are ever published on Github